webapp/packages/core/src/localization/LocalizationService.ts

```diff
@@ -44,7 +43,7 @@ export class LocalizationService {
               private settingsService: SettingsService) {
 
     // note that the default locale is always embedded in the build
-    this.localeMap.set(DEFAULT_LOCALE_NAME, defaultLocale);
+    this.loadLocaleAsync(DEFAULT_LOCALE_NAME);
```

Comment:
async operation in synchronous constructor is bad practice, I think better to move this code part to `init`

Reply:
fixed